src/main/kotlin/caculator/Calculator.kt

```diff
@@ -0,0 +1,24 @@
+package caculator
+
+class Calculator(private val delimiter: String) {
```

Comment:
인자에 대한 기본값을 제공하기 위해 부 생성자를 여럿 만들지 말고 매개 변수의 기본값을 사용하면 어떨까요?

```suggestion
class Calculator(private val delimiter: String = " ") {
```